ThingIFSDK/ThingIFSDK/ThingIFAPI.swift

```diff
@@ -656,6 +656,66 @@ open class ThingIFAPI<ConcreteAlias: Alias>: NSObject, NSCoding {
         // TODO: implement me.
     }
 
+    /** Update firmware version.
+
+     This method updates firmware version for `target` thing.
+
+     - Parameter firmwareVersion: firmwareVersion to be updated.
+     - Parameter completionHandler: A closure to be executed once on
+       checking has finished The closure takes 1 argument. The
+       argument is ThingIFError.
+     */
+    open func update(
+      firmwareVersion: String,
+      completionHandler: @escaping (ThingIFError?)-> Void) -> Void
+    {
+        // TODO: implement me.
+    }
+
+    /** Get firmeware version.
+
+     This method gets firmware version for `target` thing.
+
+     - Parameter completionHandler: A closure to be executed once on
+       checking has finished The closure takes 2 arguments. First one
+       is firmware version. Second one is ThingIFError.
+     */
+    open func getFirmewareVerson(
+      completionHandler: @escaping (String?, ThingIFError?) -> Void) -> Void
+    {
+        // TODO: implement me.
+    }
+
+    /** Update thing type.
+
+     This method updates thing type for `target` thing.
+
+     - Parameter firmwareVersion: firmwareVersion to be updated.
+     - Parameter completionHandler: A closure to be executed once on
+       checking has finished The closure takes 1 argument. The
+       argument is ThingIFError.
+     */
+    open func update(
+      thingType: String,
+      completionHandler: @escaping (ThingIFError?)-> Void) -> Void
+    {
+        // TODO: implement me.
+    }
+
+    /** Get firmeware version.
```

Comment:
@yongpingchen 

I fixed API document, but at this point, `firmware version` is correct.